### PR TITLE
Address pre-commit reported issues

### PR DESCRIPTION
**Pull request by sandrobonazzola**
addressed issues reported by pre-commit except:

```console
In create-seccomp-rules line 30:
	--arg action "$action_type" \
                      ^----------^ SC2154 (warning): action_type is referenced but not assigned.
```

reported at https://github.com/containers/qm/issues/476

### CODE REVIEW

File: tests/e2e/set-ffi-env-e2e [hunk 1]
```diff
@@ -244,9 +244,6 @@ EOF
 
 install_autosd_repo() {
     # Install autosd repository
-    source /etc/os-release
-
-    releasever=${VERSION_ID}
```
Comment (dougsland) [inline tests/e2e/set-ffi-env-e2e:249]: this remove seems incorrect as will break the line 253 (autosd.repo)

**Comment by dougsland** [comment]
@Yarboa PTAL

File: tests/e2e/set-ffi-env-e2e (hunk 1 above)
Comment (sandrobonazzola) [reply tests/e2e/set-ffi-env-e2e:249]: No, it won't. the $releasever listed in autosd.repo is going to be resolved at runtime by dnf:
```console
 /usr/libexec/platform-python -c 'import dnf, json; db = dnf.dnf.Base(); print(json.dumps(db.conf.substitutions, indent=2))'
{
  "arch": "x86_64",
  "basearch": "x86_64",
  "releasever": "9"
}
```

Comment (dougsland) [reply tests/e2e/set-ffi-env-e2e:249]: awesome, thanks sandro!

**Pull request merged**